app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/SearchAddressActivity.kt

```diff
@@ -49,8 +49,8 @@ class SearchAddressActivity : AppCompatActivity() {
                             dialog.dismiss()
                         }
                     }
-                    (resultMsg!!.obj as WebView.WebViewTransport).webView = newWebView
-                    resultMsg!!.sendToTarget()
+                    (resultMsg?.obj as WebView.WebViewTransport).webView = newWebView
```

Comment:
```suggestion
                    (resultMsg?.obj as? WebView.WebViewTransport)?.webView = newWebView
```
Type casting 또한 안전하게 처리하면 좋을 것 같아요